src/screens/QRScannerScreen.tsx

```diff
@@ -38,41 +51,142 @@ const ScannerHeader = styled(Header).attrs({
+
+    return {
+      opacity,
+    };
+  });
 
   return (
-    <View pointerEvents="box-none">
-      <ColorModeProvider value="darkTinted">
-        <Box
-          position="absolute"
-          top={{ custom: topInset - 8 }}
-          width="full"
-          style={{ zIndex: 1 }}
-        >
-          <Box alignItems="center" paddingTop="8px" height={{ custom: 16 }}>
-            <SheetHandle color="rgba(245, 248, 255, 0.4)" showBlur={false} />
+    <Box style={{ backgroundColor: colors.trueBlack }}>
+      <Box as={Animated.View} pointerEvents="box-none" style={containerStyle}>
+        <ColorModeProvider value="darkTinted">
+          <Box
+            position="absolute"
+            top={{ custom: topInset }}
+            width="full"
+            style={{ zIndex: 1 }}
+          >
+            <Navbar
+              hasStatusBarInset={false}
+              leftComponent={
+                <Navbar.Item onPress={handleCloseScanner}>
+                  <Box
+                    alignItems="center"
+                    justifyContent="center"
+                    height={{ custom: 36 }}
+                    width={{ custom: 36 }}
+                  >
+                    <Text
+                      align="center"
+                      color="label"
+                      size="icon 20px"
+                      weight="semibold"
+                    >
+                      􀆄
+                    </Text>
+                  </Box>
+                </Navbar.Item>
+              }
+              rightComponent={
+                <AccentColorProvider color="#FFDA24">
+                  <Navbar.Item onPress={() => setFlashEnabled(!flashEnabled)}>
+                    <Box
+                      alignItems="center"
+                      justifyContent="center"
+                      height={{ custom: 36 }}
+                      width={{ custom: 36 }}
+                    >
+                      <Text
+                        align="center"
+                        color={flashEnabled ? 'accent' : 'label'}
+                        size="icon 20px"
+                        weight="semibold"
+                      >
+                        {flashEnabled ? '􀞋' : '􀝌'}
+                      </Text>
+                    </Box>
+                  </Navbar.Item>
+                </AccentColorProvider>
+              }
+              title="Scan to Connect"
+            />
           </Box>
-          <Navbar hasStatusBarInset={false} title="Scan to Connect" />
-        </Box>
-        <ScannerContainer>
-          <Background />
-          <CameraDimmer cameraVisible={true}>
-            {android && (
+          <ScannerContainer>
+            <Background />
+            <CameraDimmer cameraVisible={true}>
+              {android && (
+                <ScannerHeader>
+                  <EmulatorPasteUriButton />
+                </ScannerHeader>
+              )}
+              {true && (
```

Comment:
why this condition???

Reply:
@skylarbarrera assuming this is good to remove?
![image](https://github.com/rainbow-me/rainbow/assets/6868432/7fc3ae4e-7c59-4154-bde6-5a6408a1a78c)

Reply:
yeah just revert this change